src/mongoose_client_api/mongoose_client_api_rooms_config.erl

```diff
@@ -43,8 +43,12 @@ from_json(Req, State = #{was_replied := true}) ->
 from_json(Req, State) ->
     Method = cowboy_req:method(Req),
     {ok, Body, Req2} = cowboy_req:read_body(Req),
-    JSONData = jiffy:decode(Body, [return_maps]),
-    handle_request(Method, JSONData, Req2, State).
+    case mongoose_client_api:json_to_map(Body) of
+      {ok, #{<<"name">> := N, <<"subject">> := S} = JSONData} when is_binary(N), is_binary(S) ->
```

Comment:
Yet another block of code which could benefit from the 4 spaces indentation.

Reply:
Done. Thanks. And also find few places :-)